src/gasPriceOracle/adapters/linea-viem.ts

```diff
@@ -15,10 +15,12 @@ import { fixedPointAdjustment } from "../../utils";
  * @dev Because the Linea priority fee is more volatile than the base fee, the base fee multiplier will be applied
  * to the priority fee.
  * @param provider Viem PublicClient
- * @param _chainId Unused in this adapter
+ * @param {GasPriceEstimateOptions} opts Relevant options for Linea are baseFeeMultiplier and unsignedTx.
```

Comment:
Should be similar to above
```suggestion
 * @param opts Relevant options for Linea are baseFeeMultiplier and unsignedTx.
```